go.mod

```diff
@@ -5,34 +5,34 @@ require (
 	github.com/getkin/kin-openapi v0.111.0
 	github.com/google/go-cmp v0.5.9
 	github.com/hashicorp/go-hclog v1.4.0
-	github.com/hashicorp/go-plugin v1.4.8
+	github.com/hashicorp/go-plugin v1.4.9
 	github.com/hashicorp/go-version v1.6.0
-	github.com/hashicorp/hc-install v0.4.0
-	github.com/hashicorp/hcl/v2 v2.15.0
-	github.com/hashicorp/terraform-exec v0.17.3
-	github.com/hashicorp/terraform-json v0.14.0
+	github.com/hashicorp/hc-install v0.5.0
+	github.com/hashicorp/hcl/v2 v2.16.2
+	github.com/hashicorp/terraform-exec v0.18.1
+	github.com/hashicorp/terraform-json v0.16.0
 	github.com/hashicorp/terraform-plugin-docs v0.13.0
-	github.com/hashicorp/terraform-plugin-go v0.14.2
-	github.com/hashicorp/terraform-plugin-log v0.7.0
+	github.com/hashicorp/terraform-plugin-go v0.15.0
+	github.com/hashicorp/terraform-plugin-log v0.8.0
 	github.com/hashicorp/terraform-plugin-mux v0.7.0
-	github.com/hashicorp/terraform-plugin-sdk/v2 v2.24.1
+	github.com/hashicorp/terraform-plugin-sdk/v2 v2.26.1
 	github.com/jinzhu/copier v0.3.5
 	github.com/mitchellh/go-homedir v1.1.0
 	github.com/mitchellh/hashstructure v1.1.0
 	github.com/robfig/cron v1.2.0
 	github.com/stretchr/testify v1.8.1
-	golang.org/x/mod v0.7.0
+	golang.org/x/mod v0.8.0
 	k8s.io/api v0.25.5
 	k8s.io/apiextensions-apiserver v0.25.5
 	k8s.io/apimachinery v0.25.5
 	k8s.io/client-go v0.25.5
 	k8s.io/kube-aggregator v0.25.5
 	k8s.io/kubectl v0.25.5
-	k8s.io/kubernetes v1.25.5
+	k8s.io/kubernetes v1.15.0-alpha.0
 )
```

Comment:
Could you please revise this? It seems to be a downgrade. Thanks!

Reply:
wow, that's bad! Thanks for catching that!